### PR TITLE
Add tests

### PR DESCRIPTION
**Pull request by Globoxx**
# PR Details

## Types of changes

- [ ] Docs change / refactoring / dependency upgrade
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: app/src/main/java/ch/epfl/sweng/zuluzulu/Utility/Utils.java [hunk 1]
```diff
@@ -95,7 +95,7 @@ static long getMillisecondsSince(Date date) {
      * @param container The container view
      */
     static void showConnectSnackbar(View container) {
-        Snackbar.make(container, "Login to access more features ⭐", 5000).show();
+        Snackbar.make(container, "Connecte-toi pour avoir accès à toutes les options", 5000).show();
```
Comment (Dahnepfl) [inline app/src/main/java/ch/epfl/sweng/zuluzulu/Utility/Utils.java]: You should put this in Strings.xml

Comment (Globoxx) [reply app/src/main/java/ch/epfl/sweng/zuluzulu/Utility/Utils.java]: Right

Comment (Dahnepfl) [reply app/src/main/java/ch/epfl/sweng/zuluzulu/Utility/Utils.java]: https://developer.android.com/guide/topics/resources/string-resource


https://developer.android.com/guide/topics/resources/string-resource#formatting-strings

Comment (Dahnepfl) [reply app/src/main/java/ch/epfl/sweng/zuluzulu/Utility/Utils.java]: A Snackbar is a kind on xml "textview" that you fill with text.

Comment (Globoxx) [reply app/src/main/java/ch/epfl/sweng/zuluzulu/Utility/Utils.java]: Dans ce cas on peut passer au crible tout le reste du projet parce que y'a des strings comme ça un peu partout

**Pull request merged**